dbld/rules

```diff
@@ -8,13 +8,14 @@ BUILDER_IMAGES=		\
 	debian-buster	\
 	debian-sid	\
 	ubuntu-xenial	\
-	ubuntu-bionic \
-	ubuntu-focal \
-	kira
+	ubuntu-bionic	\
+	ubuntu-focal	\
+	kira		\
+	tarball
 
 IMAGES=${BUILDER_IMAGES} devshell
 DEFAULT_IMAGE ?= ubuntu-bionic
-TARBALL_IMAGE ?= ubuntu-bionic
+TARBALL_IMAGE ?= tarball
```

Comment:
This topic come up in #3461, but why do we allow to build the tarball on other platforms, then the TARBALL_IMAGE?
I mean the `dbld/rules tarball-<platform>` command.
`		$(DOCKER) run $(DOCKER_RUN_ARGS) --rm -i balabit/syslog-ng-$* /dbld/tarball; \	`

Shouldn't we limit that tarball can be run only our tarball image?

Reply:
The target we are promoting for building tarballs is `./dbld/rules tarball` as per the help screen of the rules file, where the name of the platform to use is implied. The qualified name `tarball-<OS>` is useful as an internal target, not to mention the fact that `tarball` is just using the same conventions all of our other targets.

All in all, I don't think it hurts, users should not use it directly, but if they do that's up to them.